src/tiny_slam/tiny_grid_cells.h

```diff
@@ -1,24 +1,61 @@
+/*!
+ * \file
+ * \brief Definition of classes file: BaseTinyCell, AvgTinyCell (are inherited from GidCell), TinyBaseCellFactory, TinyAvgCellFactory (from GridCellFactory)
+ *
+ * Two classes from this file (inherited from GridCell) presents two different ways to calculate a probability of cell occupation.
+ * Two another ones (inherited from GridCellFactory)presents factories to produce cells meant before/
+ */
+
 #ifndef __TINY_GRID_CELLS_H
 #define __TINY_GRID_CELLS_H
 
 #include "../core/maps/grid_map.h"
 
 //------------------------------------------------------------------------------
 // Base cell
-
+/*!
+ * \brief Class for calculated probability with way presented in article
+ *
+ * There could be generated grid cells with simple rule of calculating probability:
+ * \f[p_i=(1-\alpha)\cdot p_{i-1}+\alpha\cdot p_{new}\f]
+ */
 class BaseTinyCell : public GridCell {
 public:
+  /*!
+   * Constructor sets the value of probability \f$p_0=0.5\f$
+   */
   BaseTinyCell(): _prob(0.5) {}
+  /*!
+   * Getter for value of probability \f$p_i\f$
+   * \return current value \f$p_i\f$ of current cell
+   */
   double value() const override { return _prob; }
+  /*!
+   * Modifier which updates value \f$p_i\f$ using previous value \f$p_{i-1}\f$, new probability value \f$p_{new}\f$ and input quality of experiment value\f$\alpha\f$
```

Comment:
Drop the "modifier which".
Consider: "Merges a given probability with already stored one using quality as a weight". The repeated formulae definition seems redundant to me.